php-plugin/src/main/java/fr/greencodeinitiative/php/checks/AvoidGettingSizeCollectionInForLoopCheck.java

```diff
@@ -19,45 +18,48 @@
         name = AvoidGettingSizeCollectionInForLoopCheck.ERROR_MESSAGE,
         description = AvoidGettingSizeCollectionInForLoopCheck.ERROR_MESSAGE,
         priority = Priority.MINOR,
-        tags = {"bug", "eco-design", "ecocode"})
+        tags = {"eco-design", "ecocode"})
```

Comment:
I think there are other tags that we can add here ... please check SonarQube existing tags and write it here (then we will discuss about your ideas)